README.md

```diff
@@ -649,3 +649,98 @@ Now you should also see the *RRbot* represented correctly in `RViz`.
    ```
 
 3. You should also see the *RRbot* moving in `RViz`.
+
+
+### Scenario showcase: Using ros2_control within a local namespace
+
+- Launch file: [rrbot_namespace.launch.py](ros2_control_demo_bringup/launch/rrbot_namespace.launch.py)
+- URDF: [rrbot.urdf.xacro](ros2_control_demo_bringup/config/rrbot.yaml)
+- ros2_control URDF: [rrbot.ros2_control.xacro](ros2_control_demo_description/rrbot_description/ros2_control/rrbot.ros2_control.xacro)
+- Controllers config: [rrbot_namespace_controllers.yaml](ros2_control_demo_bringup/config/rrbot_namespace_controllers.yaml)
+
+**NOTE:**when running `ros2 control` CLI commands you have to use additional parameter with exact controller manager node name, i.e., `-c /rrbot/controller_manager`.
+
+- Command interfaces:
+  - joint1/position
+  - joint2/position
+- State interfaces:
+  - joint1/position
+  - joint2/position
+
+Available controllers: (nodes under namespace "/rrbot")
+- `forward_position_controller[forward_command_controller/ForwardCommandController]`
+- `joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster]`
+
+List controllers:
+```
+ros2 control list_controllers -c /rrbot/controller_manager
+```
+
+Switch controller to use `position_trajectory_controller` (`JointTrajectoryController`):
+```
+ros2 control switch_controllers -c /rrbot/controller_manager --stop forward_position_controller --start position_trajectory_controller
+```
+
+Commanding the robot:
+- /rrbot/forward_position_controller:
+  ```
+  ros2 launch ros2_control_demo_bringup test_forward_position_controller.launch.py publisher_config:=rrbot_namespace_forward_position_publisher.yaml
+  ```
+
+- /rrbot/position_trajectory_controller
+  ```
+  ros2 launch ros2_control_demo_bringup test_joint_trajectory_controller.launch.py publisher_config:=rrbot_namespace_joint_trajectory_publisher.yaml
```

Comment:
Presentation thought: this doesn't output the commands as the forward_position_controller does, which may be confusing for new users. It's easy enough to echo the topic it tells you it's publishing on, but it might be a nice addition to output the commands for the demo.

Reply:
This is something we should update then in the `ros2_controllers_test_nodes` ([here](https://github.com/ros-controls/ros2_controllers/blob/master/ros2_controllers_test_nodes/ros2_controllers_test_nodes/publisher_joint_trajectory_controller.py). Is this something you would be interested to do?

Reply:
Sure! I'll create an issue and take a look this week.